custom/icds_reports/models/aggregate.py

```diff
@@ -507,6 +508,24 @@ class AggAwc(models.Model, AggregateMixin):
     cases_ccs_pregnant_all_reg_in_month = models.IntegerField(blank=True, null=True)
     cases_ccs_lactating_all_reg_in_month = models.IntegerField(blank=True, null=True)
 
+    valid_all_0_3_female = models.IntegerField(blank=True, null=True)
```

Comment:
Search about the number of columns limits in postgres. Found that if all columns are integer then the limit is 1600. else in any case minimum  it would be 250. In this table most columns are integer. I believe we would have close 1600 capacity.